02/app.js

```diff
@@ -1,3 +1,3 @@
 console.log('Działa! Gratuluję uruchomienia JS z poziomu Node.js!');
-console.log('Moja wersja Node.js to: [tutaj wpisz swoją wersję]');
-console.log('Moja wersja npm to: [tutaj wpisz swoją wersję]');
\ No newline at end of file
+console.log('Moja wersja Node.js to: v20.10.0');
+console.log('Moja wersja npm to: 10.2.3]');
```

Comment:
Wersje są ok! :)

PS. Zwróć uwagę, że to komentarz do wieli linii (szukaj numerów linii, których dotyczy)